src/libecalc/dto/models/compressor/stage.py

```diff
@@ -10,7 +10,7 @@
 class CompressorStage(EcalcBaseModel):
     compressor_chart: CompressorChart
     inlet_temperature_kelvin: confloat(ge=0)
-    pressure_drop_before_stage: confloat(ge=0)
+    pressure_drop_before_stage: Union[confloat(ge=0), str, List[float]]
```

Comment:
I don't like the `str` here. When in core and looking at this type you don't want that to be an option, it should probably be List[float] (if we think of this object as something that belongs to core).

The 'problem' is that we are using the same object in two different 'layers' I guess. I think we need the YamlCompressorStage class (currently not implemented) and another object that core expects. This conversion can happen from YamlCompressorStage to CompressorStage (domain class?). This is related to the same stuff we are trying to figure out for v2. I think we are leaning towards the dto-later being unnecessary, i.e. we should have a yaml-layer, and domain (core) objects.